drag_and_drop_v2/translations/en/LC_MESSAGES/text.po

```diff
@@ -286,10 +286,6 @@ msgid ""
 msgstr ""
 
 #: templates/html/js_templates.html
-msgid "Zones"
```

Comment:
@mtyaka submitted https://github.com/edx-solutions/xblock-drag-and-drop-v2/pull/90 - please review